### PR TITLE
bmm: add list of attempts

### PR DESCRIPTION
**Pull request by torkelrogstad**
I need some help here, @octobocto!

### CODE REVIEW

**Comment by torkelrogstad** [comment]
Screenshot 2023-11-01 at 12 43 55

How do I make the table scroll downwards when there's too many elements?

File: lib/pages/tabs/bmm_tab_page.dart [hunk 1]
```diff
@@ -79,18 +81,68 @@ class BlindMergedMiningTabPage extends StatelessWidget {
...
-                      child: viewModel.bmmResult != null
-                          ? SelectableText(
-                              viewModel.bmmResult!,
-                            )
-                          : const Text('no BMM result yet'),
+                // How do I make this scroll...
+                SingleChildScrollView(
+                  child: ConstrainedBox(
+                    constraints: const BoxConstraints(maxHeight: 300),
+                    child: DataTable(
```
Comment (torkelrogstad) [inline lib/pages/tabs/bmm_tab_page.dart:88]: this is the table in question @octobocto

File: lib/pages/tabs/bmm_tab_page.dart [hunk 2]
```diff
@@ -25,6 +30,25 @@ class BlindMergedMiningTabPage extends StatelessWidget {
...
+                    'Blind Merged Mining (BMM) is the mechanism used for mining on a sidechain.\n\n'
+                    'Pressing the "start" button here starts the BMM process. The BMM process consists '
+                    'of a sequence of attempts which can succeed or fail. When an attempt is first made '
+                    'it has a pending status. The next attempt decides the outcome of the previous one, '
+                    'which is reflected in the table below.\n\n'
+                    'A BMM attempt can also fail to be created in its entirety. In that case, it is not '
+                    "added to the table. Notably, a BMM attempt only succeeds if there's been mined at"
+                    'least 1 block on the mainchain since the last BMM attempt.\n\n'
+                    "A BMM attempt includes a satoshi denominated 'bid' amounts. This amount is paid "
+                    'the the mainchain miners for the right to construct a new sidechain block. The '
```
Comment (octobocto) [inline lib/pages/tabs/bmm_tab_page.dart:45]: hmm, seems off

**Comment by octobocto** [comment]
added two commits!

**Comment by octobocto** [comment]
And rebased, because I broke a bunch of things

File: lib/pages/tabs/bmm_tab_page.dart (hunk 2 above)
Comment (torkelrogstad) [reply lib/pages/tabs/bmm_tab_page.dart:46]: wdym, spelling mistakes? fixed those now

**Pull request merged**